main.py

```diff
@@ -64,6 +64,39 @@
 PUNISHMENT_INACTIVITY = 5
 
 
+def calculate_all_fitness(end_round_time, player_list, genomes_list):

```

Comment:
The function is called calculate_all_fitness but what does the boolean value represent. I imagine it could represent whether the game is finished or not; thus 'game_finished' could be a better function name? Then could create a new function called 'calculate_remaining_fitness' to iterate through player list and calculate the fitnesses of the players?

Thoughts? 
Also, the countdown can be removed tbh. My bad.

Reply:
Makes sense